docs/source/aequilibrae_project/project_components.rst

```diff
@@ -27,9 +27,557 @@ the items table is a ``Link`` object.
+
+    # One can also check all the fields in the 'nodes' table.
+    >>> node_fields.all_fields() # doctest: +ELLIPSIS
+    ['is_centroid', ..., 'my_new_field']
+
+    >>> project.close()
+
+All field descriptions are kept in the table 'attributes_documentation'.
+
+.. seealso::
+
+    *  :func:`aequilibrae.project.FieldEditor`
+        Class documentation
+
+``project.log``
+~~~~~~~~~~~~~~~
+
+Every AequilibraE project contains a log file that holds information on all the project procedures.
+It is possible to access the log file contents, as presented in the next code block.
+
+.. doctest::
+
+    >>> project = Project()
+    >>> project.open("/tmp/accessing_nauru_data")
+
+    >>> project_log = project.log()
+
+    # Returns a list with all entires in the log file.
+    >>> print(project_log.contents()) # doctest: +ELLIPSIS
+    ['2021-01-01 15:52:03,945;aequilibrae;INFO ; Created project on D:/release/Sample models/nauru', ...]
+
+    # If your project's log is getting cluttered, it is possible to clear it. 
+    # Use this option wiesly once the deletion of data in the log file can't be undone.
+    >>> project_log.clear()
+
+    >>> project.close()
+
+.. seealso::
+    
+    * :func:`aequilibrae.project.Log`
+        Class documentation
+    * :ref:`useful-log-tips`
+        Usage example
+    
+``project.matrices``
+~~~~~~~~~~~~~~~~~~~~
+
+This method is a gateway to all the matrices available in the model, which allows us to update the
+records in the 'matrices' table. Each item in the 'matrices' table  is a ``MatrixRecord`` object.
+
+.. doctest::
+
+    >>> project = Project()
+    >>> project.open("/tmp/accessing_sfalls_data")
+
+    >>> matrices = project.matrices
+
+    # One can also check all the project matrices as a Pandas' DataFrame
+    >>> matrices.list() # doctest: +SKIP
+
+    # We can add a naw matrix
+    >>> matrices.new_record() # doctest: +SKIP
+    
+    # To delete a matrix from the 'matrices' table, we can delete the record directly
+    >>> matrices.delete_record("demand_mc")
+    
+    # or by selecting the matrix and deleting it
+    >>> mat_record = matrices.get_record("demand_mc") # doctest: +SKIP
+    >>> mat_record.delete() # doctest: +SKIP
+
+    # If you're unsure if you have a matrix in you project, you can check if it exists
+    # This function will return `True` or `False`
+    >>> matrices.check_exists("my_matrix")
+    False
+
+    # If a matrix was added or deleted by an external process, you should update or clean
+    # your 'matrices' table to keep your project organised.
+    >>> matrices.update_database()  # in case of addition
+    
+    >>> matrices.clear_database()  # in case of deletion
+
+    # To reload the existing matrices in memory once again
+    >>> matrices.reload()
+
+    >>> project.close()
+
+.. seealso::
+
+    * :func:`aequilibrae.project.Matrices`
+        Class documentation
+    * :ref:`matrix_table`
+        Table documentation
+
+OpenMatrix (OMX)
```

Comment:
We need to keep a page for the AequilibraE matrix class.  It is central to transport modelling the mechanics around matrices